app/src/test/java/io/intrepid/contest/screens/contestcreation/reviewcontest/ReviewContestPresenterTest.java

```diff
@@ -0,0 +1,46 @@
+package io.intrepid.contest.screens.contestcreation.reviewcontest;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import io.intrepid.contest.models.Contest;
+import io.intrepid.contest.testutils.BasePresenterTest;
+import io.intrepid.contest.testutils.TestPresenterConfiguration;
+
+import static org.mockito.Mockito.verify;
+
+public class ReviewContestPresenterTest extends BasePresenterTest {
```

Comment:
pass in the Presenter as the generic type to `BasePresenterTest`. Then you can use the `presenter` member variable defined in `BasePresenterTest`